app/views/posts/_form.html.erb

```diff
@@ -88,8 +92,9 @@
     var input= editor.getValue();
     editor2.setValue(">>>" + input);
 
+    //Run button's send()function 
     function send(){
-      window.location.href = "http://stackoverflow.com";
+      window.location.href = "https://github.com/CS121Fresh/Coconut-Interpreter";
```

Comment:
It indeed makes more sense to have the run button linked to our github repo instead of stackoverflow.com at this stage as we have not yet been able to run Python code on our website.